akka-typed-tests/src/test/scala/akka/typed/DeferredSpec.scala

```diff
@@ -79,14 +79,36 @@ class DeferredSpec extends TypedSpec {
 
     def `must stop when exception from factory`(): Unit = {
```

Comment:
ok, now I see that the new test is actually testing exception from the factory, but from the inner `deferred`, that should be enough.